src/stream/src/executor/hash_join.rs

```diff
@@ -29,6 +29,7 @@ use risingwave_common::row::{OwnedRow, Row};
 use risingwave_common::types::{DataType, ToOwnedDatum};
 use risingwave_common::util::epoch::EpochPair;
 use risingwave_common::util::iter_util::ZipEqDebug;
+use risingwave_common::util::value_encoding::ValueRowSerde;
```

Comment:
Why use the column version aware encoding?🤔

Reply:
> Why use the column version aware encoding?🤔

It's a trait bound in state table generic parameters.

Reply:
column version aware encoding is used for schema change, in general, we should use `BasicSerde`, which is the default setting for state table. Is there any specific reason to use this?

Reply:
> column version aware encoding is used for schema change, in general, we should use `BasicSerde`, which is the default setting for state table. Is there any specific reason to use this?

Hmm, but `ValueRowSerde` is a trait bound...